app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -76,7 +76,6 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
         webRenderComponents.notifyLaunchWithSafeIntent(safeIntent)
 
         lifecycle.addObserver(serviceLocator.engineViewCache)
-        lifecycle.addObserver(serviceLocator.pocketVideoFetchScheduler)
```

Comment:
Can we also remove the following classes in their entirety?
- PocketVideoFetchScheduler
- PocketVideoFetchWorker
- PocketVideoStore
- PocketVideoJSONValidator
- PocketEndpoint
- PocketVideoRepo
- PocketViewModel
- ? any I missed?

And also the WorkManager gradle dependency which I believe is not needed anymore.

I'm concerned that leaving these in – particularly the fetching and the store – leaves the possibility that there is still a dangling reference that it will be used and function in a way we don't expect.